### DISCUSSION TITLE
webhook using python flask

### DISCUSSION THREAD

**Discussion by ajuroshan** [vasani-arpit/WBOT#339] [Q&A]
Here's the webhook server.
It just returns the received json as it is.

![image](https://user-images.githubusercontent.com/70354172/209809648-b4ac9678-67d6-4c34-8f90-89858eca9a51.png)

Here's bot.json

![image](https://user-images.githubusercontent.com/70354172/209809743-8ff3b40c-aa77-4391-9081-1154ef3f86c9.png)

I don't know why this happens. I've been stuck on this for two days. 

![image](https://user-images.githubusercontent.com/70354172/209810041-5ed3e8bb-a676-48d9-ae26-2244fc08561d.png)

python Log

![image](https://user-images.githubusercontent.com/70354172/209810237-52979039-fecf-484d-b8f6-c227f1882d33.png)

Any help ?

**Comment by vasani-arpit** [comment]
have you looked at this?
https://github.com/vasani-arpit/WBOT/blob/master/docs/Webhook-example.md
